src/app/country/country.component.html

```diff
@@ -1,8 +1,21 @@
 <main class="fit-to-page">
   <h1 class="page-title">
     <span class="flag-orb" [ngClass]="isoCountryCodes[country]"></span>
-    <span>{{ country }}</span>
-    <span *ngIf="country" i18n="@@countryPageTitle">Data</span>
+    <ng-container
+      *ngIf="
+        headerRef &&
+        headerRef['countryTotalMap'] &&
+        headerRef['countryTotalMap'][country] &&
+        headerRef['countryTotalMap'][country]['percentage'] !== undefined
+      "
+    >
+      <span *ngIf="country">{{ country }}</span>
+      <span *ngIf="country" i18n="@@countryPageTitle">Data</span>
```

Comment:
Should these two lines not always be shown (regardless of the existence of the country percentage)? Otherwise you'd theoretically have a page with no title, wouldn't you?

Reply:
The condition here is whether or not the country is set - and it will ALWAYS be set - but because it's not set until the `url` parameter is read, and because that happens within a combined pipeline that's also loading data, there's a "clunk" between the word data showing and then it realigning when the country name drops in.  The purpose of this condition is entirely aesthetic.

Reply:
So there is never the situation where a country IS set, but the percentage IS NOT set?

Reply:
That situation cannot happen.